### PR TITLE
Manar_Pull_Request

### PR DESCRIPTION
**Pull request by ManarMahmoud17**
To implement functionality for the "Delete" and "Check" buttons using the MVC (Model-View-Controller) principle

### CODE REVIEW

**Review by rawanramadanseddik — APPROVED** [review]
Everything is good, "I think"

**Review by youssefibrahim22 — COMMENTED** [review]
just remove the script.js file as its not needed

**Review by MiraWassem — APPROVED** [review]
PERFECTOOOO

File: script.js
Comment (Rubazy) [inline script.js:1]: This file is useless, so it is better to remove it.

File: index.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    const todo = this.todos.find(todo => todo.id === id);
+    if (todo) {
+      todo.completed = !todo.completed;
+      view.updateTodoStatus(id, todo.completed);
+    }
+  },
+};
+
+const view = {
+  todoListElem: null, // Define the property
```
Comment (Rubazy) [inline index.js]: what is the purpose of this comment?

**Review by Rubazy — APPROVED** [review]

**Pull request merged**